### PR TITLE
Advancing 0.14.0-m0 release to status: projects

### PR DESCRIPTION
**Pull request by maayanf24**
Advancing 0.14.0-m0 release to status: projects

### CODE REVIEW

File: releases/v0.14.0-m0.yaml [hunk 1]
```diff
@@ -2,7 +2,10 @@
 version: v0.14.0-m0
 name: 0.14.0-m0
 pre-release: true
-status: admiral
+status: projects
 components:
   shipyard: 4d1252c3148034c2da19e53304a266972195ba27
   admiral: 069480a1e9b18d82f681b3f5ed673a442a7aef11
+  cloud-prepare: f28f1db35bc3188bdff1ad1cae682ffbf02123e4
```
Comment (dfarrell07) [inline releases/v0.14.0-m0.yaml:9]: https://github.com/submariner-io/cloud-prepare/commit/f28f1db35bc3188bdff1ad1cae682ffbf02123e4

File: releases/v0.14.0-m0.yaml [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+  lighthouse: 0eefcd98c4385df3b403e9d092458efaf83b5ea2
```
Comment (dfarrell07) [inline releases/v0.14.0-m0.yaml:10]: https://github.com/submariner-io/lighthouse/commit/0eefcd98c4385df3b403e9d092458efaf83b5ea2

File: releases/v0.14.0-m0.yaml [hunk 3]
```diff
@@ -2,7 +2,10 @@
...
+  submariner: 1a8dab3f2eb2e7869881179788e076f2d240d743
```
Comment (dfarrell07) [inline releases/v0.14.0-m0.yaml:11]: https://github.com/submariner-io/submariner/commit/1a8dab3f2eb2e7869881179788e076f2d240d743

**Review by dfarrell07 — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Comment by dfarrell07** [comment]
> [14:28:01.441] [releases]$ docker pull quay.io/submariner/subctl:devel-ff63dab
Error response from daemon: manifest for quay.io/submariner/subctl:devel-ff63dab not found: manifest unknown: manifest unknown

**Comment by tpantelis** [comment]
> > [14:28:01.441] [releases]$ docker pull quay.io/submariner/subctl:devel-ff63dab
> > Error response from daemon: manifest for quay.io/submariner/subctl:devel-ff63dab not found: manifest unknown: manifest unknown

Needed to wait for the **subctl** release jobs to finish.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**